src/cryptography/hazmat/primitives/asymmetric/types.py

```diff
@@ -37,27 +46,51 @@
     x25519.X25519PrivateKey,
     x448.X448PrivateKey,
 ]
+PRIVATE_KEY_TYPES = PrivateKeyTypes
+utils.deprecated(
+    PRIVATE_KEY_TYPES,
+    __name__,
+    "Use PrivateKeyTypes instead",
+    utils.DeprecatedIn40,
+    name="PRIVATE_KEY_TYPES",
+)
 # Just the key types we allow to be used for x509 signing. This mirrors
 # the certificate public key types
-CERTIFICATE_PRIVATE_KEY_TYPES = typing.Union[
+CertificatePrivateKeyTypes = typing.Union[
```

Comment:
I found this counter-intuitive. Unless somebody chimes in, I'm inclined to actually rename this to `CertificateIssuerPrivateKeyTypes`, because it mirrors `CertificateIssuerPublicKeyTypes`, and *not* `CertificatePublicKeyTypes` (this latter one allows X25519PublicKey, X448PublicKey)

Reply:
That makes sense to me.